openpype/hosts/nuke/api/lib.py

```diff
@@ -3423,3 +3423,55 @@ def create_viewer_profile_string(viewer, display=None, path_like=False):
     if path_like:
         return "{}/{}".format(display, viewer)
     return "{} ({})".format(viewer, display)
+
+
+def get_head_filename_without_hashes(original_path, name):
+    """Function to get the renamed head filename without frame hashes
+    To avoid the system being confused on finding the filename with
+    frame hashes if the head of the filename has the hashed symbol
+
+    Examples:
+        >>> get_head_filename_without_hashes("render.####.exr", "baking")
+        render.baking.####.exr
+        >>> get_head_filename_without_hashes("render.%d.exr", "tag")
+        render.tag.%d.exr
+        >>> get_head_filename_without_hashes("exr.####.exr", "foo")
+        exr.foo.%04d.exr
```

Comment:
This example is wrong.
```suggestion
        >>> get_head_filename_without_hashes("exr.%04d.exr", "foo")
        exr.foo.%04d.exr
```